src/components/NewMeasure/CreateNewMeasureDialog.tsx

```diff
@@ -200,18 +211,26 @@ const CreateNewMeasureDialog = ({ open, onClose }) => {
             "aria-describedby": "measureName-helper-text",
             required: true,
           }}
-          helperText={formikErrorHandler("measureName", true)}
+          helperText={
+            (formik.touched["measureName"] || focusedField === "measureName") &&
+            (formikErrorHandler("measureName", true) ||
+              "Measure Library name must start with an upper case letter, followed by alpha-numeric character(s) and must not contain spaces or other special characters.")
+          }
```

Comment:
I was unable to render this text while reviewing. Instead, I see "A Measure Name must contain at least one letter." when attempting to use "3" as input. Lowercase, spaces, and special characters did not trigger the expected helper text.